Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1564,6 +1564,7 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         let miscResources = Set(bundle.miscResourceURLs)
         try assetManagers[bundle.identifier, default: DataAssetManager()]
             .register(data: miscResources, dataProvider: dataProvider, bundle: bundle)
+        fallbackAssetResolvers[bundle.identifier] = ExternalMediaResolver()
```

Comment:
Thank you. This resolves my previous feedback about potentially surprising behaviors with default values for fallback resolvers.

Reply:
Currently this will break one test case(testConvertPageWithLinkResolving) since we will set fallbackAssetResolvers when linkResolvingServer is not nil.

https://github.com/apple/swift-docc/blob/783cc969a0185373c92eb864c04c51b689cc978a/Sources/SwiftDocC/DocumentationService/Convert/ConvertService.swift#L148-L158

Should we move the external image functionality into a default behavior instead of injecting it into fallbackAssetResolvers? 👇
Which means we move `ExternalMediaResolver.swift`'s resolve logic into `resolveAsset(named name: String, in parent: ResolvedTopicReference)` directly.

https://github.com/apple/swift-docc/blob/783cc969a0185373c92eb864c04c51b689cc978a/Sources/SwiftDocC/Infrastructure/DocumentationContext.swift#L2625-L2640

(Add the following logic into L2630 instead of `ExternalMediaResolver.swift` and registering it as a FallbackAssetResolver)
```swift
    func resolve(assetNamed assetName: String, bundleIdentifier: String) -> DataAsset? {
        guard let externalMediaLink = URL(string: assetName),
              externalMediaLink.isAbsoluteWebURL else {
            return nil
        }
        var asset = DataAsset()
        asset.context = .display
        asset.register(externalMediaLink, with: DataTraitCollection(userInterfaceStyle: .light, displayScale: .standard))
        return asset
    }
```

Reply:
> Should we move the external image functionality into a default behavior instead of injecting it into fallbackAssetResolvers? 👇
> Which means we move ExternalMediaResolver.swift's resolve logic into resolveAsset(named name: String, in parent: ResolvedTopicReference) directly.

That sounds good to me 👍